{"name": "Проверка логина c неправильным паролем", "status": "passed", "description": "Пытаемся залогиниться в аккаунт курьера, получаем ошибку 404 и соотвествующее сообщение", "start": 1721375125622, "stop": 1721375125705, "uuid": "8ea15a3f-f450-45b0-904e-210bdf9c87bf", "historyId": "59c1190257832c75b2535cb60e758ffd", "testCaseId": "59c1190257832c75b2535cb60e758ffd", "fullName": "tests.test_courier_login.TestLoginCourier#test_no_such_username_and_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "LAPTOP-NE0EJ4CA"}, {"name": "thread", "value": "10612-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}